TG.Backend/Controllers/AuthController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using TG.Backend.Filters;
+
+namespace TG.Backend.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class AuthController : ControllerBase
+    {
+        private readonly IMediator _mediator;
```

Comment:
How about using ISender instead of IMediator interface ? We do not publish notifications so ISender would be enoguh :)

Reply:
Sure, that is a good idea, I will resolve this issue soon.